frontend/src/components/ui/copy-button.ts

```diff
@@ -61,7 +61,7 @@ export class CopyButton extends LitElement {
         @sl-after-hide=${this.stopProp}
       >
         <sl-icon-button
-          name=${this.isCopied ? "check-lg" : this.name ? this.name : "files"}
+          name=${this.isCopied ? "check-lg" : this.name ? this.name : "copy"}
```

Comment:
We could also switch to the [shoelace copy button](https://shoelace.style/components/copy-button)

Reply:
Heh my [one contribution](https://github.com/shoelace-style/shoelace/commit/8ddef1a0bd478929074b94a80921796808be82d6) in the Shoelace repo was changing _that_ icon too :)

Right now it's still tagged as "experimental" though I don't know what breaking changes would be made to it...  Would be in favour of replacing our copy button with theirs.